project.md

```
@@ -0,0 +1,138 @@
+    file_path = f"{UPLOAD_DIR}/{filename}"
+    if not os.path.exists(file_path):
+       return {"error": "File not found"}
+    try:
+       doc = Document(file_path)
+       text = "\n".join([para.text for para in doc.paragraphs])
+       return {"filename": filename, "content": text}
+    except Exception as e:
+       return {"error": str(e)}
+```
+
+3. Run backend:
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)